core/ecschema-editing/src/Editing/Properties.ts

```diff
@@ -17,7 +17,7 @@ type MutablePropertyType = MutableProperty | MutableArrayProperty | MutablePrimi
  * A class allowing editing of attributes of the base Property class.
  */
 export class Properties {
-  public constructor(protected _schemaEditor: SchemaContextEditor) { }
+  public constructor(protected ecClassType: SchemaItemType, protected _schemaEditor: SchemaContextEditor) { }
```

Comment:
Shouldn't we restrict to `ECClassSchemaItems` instead of SchemaItemType type here?

Reply:
Yeah, I created the type and forgot to use it!  It wasn't being used anywhere. Done.